pkg/controller/installation/core_controller.go

```diff
@@ -1460,6 +1474,21 @@ func (r *ReconcileInstallation) checkActive(log logr.Logger) (*corev1.ConfigMap,
 	}
 }
 
+func (r *ReconcileInstallation) updateCRDs(ctx context.Context, variant operator.ProductVariant, log logr.Logger) error {
+	if !r.manageCRDs {
```

Comment:
I wonder if this check should be done by the caller, looking at the calling code it looks like we're updating the CRD's no matter what.

Reply:
Do you think we need to know when looking through the Reconcile function if we actually do update the CRDs or not?
I thought it was reasonable to hide that decision, especially since nothing else in the Reconcile cares if we've updated the CRDs or not.

Reply:
Ya that's fair, having this here makes sense as well.